WebUI/lims/shared.py

```diff
@@ -74,3 +74,19 @@ def visitor_ip_address(request):
     else:
         ip = request.META.get('REMOTE_ADDR')
     return ip
+
+'''
+[]	A set of characters
+\	Signals a special sequence (can also be used to escape special characters)
+.	Any character (except newline character)
+^	Starts with	
+$	Ends with	
+*	Zero or more occurrences
++	One or more occurrences
+{}	Exactly the specified number of occurrences
+|	Either or	
+()	Capture and group
+'''
+def regCheck(regularExp,stringIn):
+    result = re.search(regularExp, stringIn)
```

Comment:
It doesn't seem like this function is really doing much that just directly calling re.search is. I'd recommend removing the function and just directly calling re.search in the conditional.

Reply:
Ok, I will change this.